src/shared/directives/conditional-show/domain/conditional-show.model.spec.ts

```diff
@@ -350,7 +351,7 @@ describe('conditional-show', () => {
     });
 
     it('field mentioned in condition has no value asked in CONTAINS condition', () => {
-      let sc = new ShowCondition('fieldCONTAINS"test,mest"');
+      let sc = new ShowCondition('field CONTAINS"test,mest"');
```

Comment:
Same as the comment above about operators not needing to have spaces before or after...

Reply:
As I mentioned in my earlier comment